docs/DeveloperGuide.md

```diff
@@ -805,14 +803,19 @@ Pre-condition: The student's contact currently exists in TutorAid.
 
 **Extensions**
 
-* 3a. TutorAid detects that the command keyed in is incorrect. (e.g. wrong format, missing arguments, invalid indices)
+* 3a. TutorAid detects that the command keyed in is incorrect. (e.g. wrong format, missing arguments, invalid indices).
     * 3a1. TutorAid displays an error message and requests the tutor to re-enter the command.
     * 3a2. Tutor re-enters the command, along with the necessary arguments.
 
 Steps 3a1-3a2 are repeated until the command entered is correct.
 
 Use case resumes from Step 4.
 
+* 4a. TutorAid detects that the student does not have any progress note.
```

Comment:
I am wondering if this warning case is needed as will this be considered as part of Extension 3a (because if there are no notes, entering something like `del -p 1` will result in `1` being considered as an invalid index)

Reply:
The index is a student index, not for progress. Unless what you are saying is if the student has no progress entry then the student index is invalid?